src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs

```diff
@@ -18,6 +18,11 @@ public LoggerPhaseScope(string phaseName)
         }
 
         public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
+            : this(phaseName, enablePerformanceScope, LogLevel.Diagnostic)
+        {
+        }
+
+        public LoggerPhaseScope(string phaseName, bool enablePerformanceScope, LogLevel perfLogLevel)
```

Comment:
>bool enablePerformanceScope,  [](start = 50, length = 29)

remove, if use LoggerPhaseScope(phaseName, perfLogLevel), perf log must be true.